src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -22,6 +22,43 @@
 from run_utils import run_remote
 
 
+def cleanup_mounted_path(test, hosts, path):
+    """cleanup mounted paths.
+
+    Args:
+        test (Test): the test which created the mount points
+        hosts (NodeSet): hosts from which to unmount the paths
+        path (str): the mounted posix test path to cleanup
+
+    Returns:
+        list: a list of any errors detected when removing the pool
+    """
+    # need to remove contents before umount
+    error_list = cleanup_path(test, hosts, path)
+    result = run_remote(test.log, hosts, f"sudo umount -f '{path}'")
+    if not result.passed:
```

Comment:
Need to unmount and then remove

Reply:
We need a `cleanup_path(test, hosts, f'path/*'`) call, umount, and then a `cleanup_path(test, hosts, path)` call.